app/BUILD.bazel

```diff
@@ -67,7 +67,10 @@ LISTENERS = [
 DATABINDING_LAYOUTS = ["src/main/res/layout*/**"]
 
 # TODO(#1617): Remove genrules post-gradle
-# Files altered by genrules to be excluded in the app library
+'''
+View model files to be processed by genrules. View models should be processed if they import
```

Comment:
Maybe frame this & others as: "View models that import resources or binding files must be included in this list"

Reply:
Leaving the new documentation from an older PR here - let me know if you would like me to change it.